tests/Run-Tests.ps1

```diff
@@ -186,7 +190,7 @@ function ExecTest($argList)
 if ($TestDirectory -eq "")
 {
     $MyPath = (get-item $MyInvocation.MyCommand.Path).Directory.FullName;
-    $TestSrcDirectory = Join-Path $MyPath "..\build\$Platform\$Config\Tests"
+    $TestSrcDirectory = Join-Path $MyPath "..\build\$Platform\$Config\Universal Windows\"
```

Comment:
yep! no more copies of stuff. Its one flat output directory

---
In reply to: [102312399](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102312399) [](ancestors = 102312399)